ComputerVision/CIFAR/scripts/launch_pretraining.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+seed=${seed:-0}
+lr=${learning_rate:-"0.1"}
+model=${model:-"resnet_18"}
+data=${data:-"cifar100"}
+epochs=${epochs:-"200"}
+batch_size=${batch_size:-"128"}
+log_interval=${log_interval:-"100"}
+weight_decay=${weight_decay:-"5e-4"}
+output_dir=${output_dir:-"/cb/home/vithu/extra-storage/want/resnet18"}
+num_steps=${num_steps:-1}
+block_name=${block_name:-"Block"}
+optimizer=${optimizer:-"sgd"}
+momentum=${momentum:-"0.9"}
+
+# Sparsity
+############
+maskopt=${maskopt:-"None"}
+maskopt_begin_iteration=${maskopt_begin_iteration:-0}
+maskopt_frequency=${maskopt_frequency:-100}
+maskopt_sparsity=${maskopt_sparsity:-0.5}
+maskopt_mask_init_method=${maskopt_mask_init_method:-"random"}
+maskopt_mask_distribution=${maskopt_mask_distribution:-"uniform"}
+flop_budget=${flop_budget:-"1115226800"} # budget for ResNet18 v1.5
+fused_bn=${fused_bn:-"false"}
+enable_head=${enable_head:-"false"}
+enable_stem=${enable_stem:-"false"}
+
+# sift params
+sift_scaling=${sift_scaling:-"false"}
+sift_family=${sift_family:-"sparse_wide"}
+sp_use_internal_relu6=${sp_use_internal_relu6:-"false"}
+sp_use_layer_specialization=${sp_use_layer_specialization:-"false"}
+sparse_u_block=${sparse_u_block:-"false"}
+uv_layer=${uv_layer:-"None"}
+sparse_width_scaling=${sparse_width_scaling:-0.0}
+
+# Build the command
+############
+CMD="/cb/home/vithu/anaconda3/envs/want/bin/python -u main.py "
```

Comment:
let's just set this to `python -u main.py` ?

Reply:
done